proposed/http-middleware/middleware.md

```diff
@@ -46,7 +45,7 @@ object that implements the `MiddlewareInterface`.
 An HTTP middleware dispatcher is an object that holds multiple middleware
 components that can be used to process one or more requests in sequence.
 
-The middleware dispatcher MUST pass the request and a delegate to each
+The middleware dispatcher MUST pass the request and a delegate handler to each
```

Comment:
This is now the first time the term `delegate` crops up. I suggest rephrasing from "delegate handler" to "request handler". The following line would then replace "delegate" as well.